content/docs/dvclive/ml-frameworks/catalyst.md

```diff
@@ -28,8 +28,19 @@ runner.train(
 +    callbacks=[DvcLiveCallback()])
 ```
 
-This will generate the outputs as described in the
-[Get Started](/docs/dvclive/get-started#outputs).
+The [history](/doc/dvclive/api-reference/live/log#step-updates) of each
+`{metric}` will be stored in:
+
+```py
+{Live.dir}/scalars/{split}/{metric}.tsv
+```
+
+Where:
+
+- `{Live.dir}` is the
+  [`dir` attribute of `Live`](/doc/dvclive/api-reference/live#attributes).
+- `{split}` can be either `train` or `eval`.
+- `{metric}` is the name provided by the framework.
```

Comment:
Where does this catalyst standardization come from? I don't see it in https://github.com/iterative/dvclive/pull/259/files.

Reply:
Its the default format used by the framework

Reply:
😓 Catalyst uses `valid` instead of `eval` so I forgot to include it in the standardization